src/utils/file-processor.js

```diff
@@ -157,8 +159,11 @@ const sendToAlphaBroker = (byteChunks, file, handle) =>
 
 const sendToBetaBroker = (byteChunks, file, handle) =>
   new Promise((resolve, reject) => {
-    sendFileContentsToBroker(file, handle, byteChunks.reverse(), byteLocation =>
-      Math.min(file.size, byteLocation + CHUNK_BYTE_SIZE)
```

Comment:
Let's change the byte chunk size up from 30 and change the comment about byte size in this file?

Reply:
Sure I'll change it to 1000. What comment are you referring to?

Reply:
Just this one where it specifies the chunk sizes

const createByteChunks = file => {
  // This returns an array with the starting byte pointers
  // ex: For a 150 byte file it would return: [0, 31, 62, 93, 124]

Reply:
👍 updated